libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,164 @@
+                let oldDate = value;
+                if (!oldDate) {
+                    oldDate = moment(defaultTime).toDate();
+                }
+
+                let updatedDatetime = moment(selectedDate).hours(oldDate.getHours());
+                updatedDatetime = moment(updatedDatetime).minutes(oldDate.getMinutes());
+
+                // set default time if necessary && configured
+                if (updatedDatetime.isSame(selectedDate)) {
+                    const time = moment(defaultTime, TIME_FORMAT).toDate();
+                    updatedDatetime.hours(time.getHours());
+                    updatedDatetime.minutes(time.getMinutes());
+                }
+
+                selectedDate = updatedDatetime.toDate();
+            }
+
+            onChange(selectedDate);
+        };
+
+        const onTimeChange = (input: string) => {
+            let date = value;
+            if (!date) {
+                // set today in case of invalid date
+                date = new Date();
+            }
+
+            const time = moment(input, TIME_FORMAT);
+            if (time.isValid()) {
+                date.setHours(time.hours());
+                date.setMinutes(time.minutes());
+            }
+
+            onChange(date);
+        };
+
+        return (
+            <div className={cx(className, isTimeForStaticPeriodEnabled && "gd-flex-row")}>
+                {isMobile ? (
+                    <DateRangePickerInputFieldBody
+                        type="date"
+                        className={cx(
+                            "s-date-range-picker-date",
+                            "gd-date-range-picker-input",
+                            "gd-date-range-picker-input-native",
+                            error && "gd-date-range-picker-input-error",
+                        )}
+                        placeholder={placeholderDate}
+                        onChange={(event) =>
+                            onDateChange(convertPlatformDateStringToDate(event.target.value))
+                        }
+                        value={getFormattedDate()}
+                    />
+                ) : (
+                    <DateRangePickerInputField
+                        className={cx(
+                            `s-date-range-picker-date`,
+                            error && "gd-date-range-picker-input-error",
+                        )}
+                        classNameCalendar={`s-date-range-calendar`}
+                        ref={ref}
+                        onDayChange={onDateChange}
+                        value={value}
+                        format={dateFormat}
+                        placeholder={placeholderDate}
+                        dayPickerProps={{
+                            ...dayPickerPropsWithDefaults,
+                            onDayClick: handleDayClick,
+                        }}
+                        // showOverlay={placeholderDate ==="to"} // Always shows the calendar, useful for CSS debugging
```

Comment:
Does this still work?

Reply:
adjusted to be more generic `showOverlay={true}`. Feature is still supported.